pom.xml

```diff
@@ -149,6 +193,9 @@
     </profiles>
 
     <properties>
+        <jcommander.version>1.72</jcommander.version>
+        <slf4j.version.range>[1.7.0,1.8.0)</slf4j.version.range>
+        <org.json.version>20180130</org.json.version>
```

Comment:
There is a newer version: https://search.maven.org/artifact/org.json/json/20180813/bundle